src/resolvers/Security.js

```diff
@@ -70,6 +71,9 @@ export default {
   },
 
   async stethoscopeVersion (root, args, context) {
+    if (config.allowPrerelease) {
```

Comment:
I think this could be simplified to:
`return semver.satisfies(pkg.version, args.stethoscopeVersion, { includePrerelease: config.allowPrerelease === true })`

Reply:
👍

I modified this a tiny bit since `config.allowPrerelease` is a bool.